src/jit/importer.cpp

```diff
@@ -18376,110 +18387,33 @@ void Compiler::impDevirtualizeCall(GenTreeCall*            call,
     }
 
 #if defined(DEBUG)
-
     // Bail if devirt is disabled.
     if (JitConfig.JitEnableDevirtualization() == 0)
     {
         return;
     }
 
+    const bool doPrint = JitConfig.JitPrintDevirtualizedMethods() == 1;
```

Comment:
I'm confused what you're trying to achieve with `doPrint` in release builds here... looks like you only fill in the info strings under `#if DEBUG`, so release would just print the placeholders and the only info you really get is the total number of devirtualized calls per kind, right?  If we're going to want that information from release builds, would it be better to fire an ETW event or something?